README.adoc

```diff
@@ -198,17 +171,23 @@ include::{common-includes}/mvnbuild.adoc[]
 
 When the server is running, point your browser to the
 `http://localhost:9080/inventory/systems/localhost` URL.
-You receive a result in JSON format with the system properties of your local JVM. Next, point your
-browser to the `http://localhost:9080/system/properties` URL, which retrieves the information for a
-specific host. Notice that the results from the two URLs are identical.
+You receive the system properties of your local JVM from the `inventory` service. Next, point your
+browser to the `http://localhost:9080/system/properties` URL for the `system` service,
+which retrieves the system properties for the specific `localhost`.
+Notice that the results from the two URLs are identical because `inventory` service gets its results from
+calling the `system` service.
 
-You can test the fault tolerance mechanism of your application by dynamically changing
+You can test the fault tolerance mechanism of your microservices by dynamically changing
 the `io_openliberty_guides_system_inMaintenance` property value to `true` in the
 `resources/CustomConfigSource.json` file. After saving the file, go back to your browser and
 refresh to the `http://localhost:9080/inventory/systems/localhost` URL to view the cached version of
-the properties. Point your browser to the `http://localhost:9080/system/properties` URL again, and
-notice that it is no longer available. The cached system properties contain only the OS name and
-user name key and value pairs. When you're done, change the `io_openliberty_guides_system_inMaintenance`
+the properties. The `fallbackForGet()` method is being called when the `system` service is not available.
+The cached system properties contain only the OS name and user name key and value pairs.
```

Comment:
We can still better to explain the `@Fallback` behaviour.  But, this can be a separate revision.

Reply:
Attempted a fix.